django/contrib/admin/static/admin/js/SelectFilter2.js

```diff
@@ -15,6 +15,7 @@ Requires core.js and SelectBox.js.
             const from_box = document.getElementById(field_id);
             from_box.id += '_from'; // change its ID
             from_box.className = 'filtered';
+            from_box.setAttribute('aria-labelledby', field_id + '_from_title');
```

Comment:
I have done this so the help text is also announced, as select boxes don't support `aria-describedby` very well 
`aria-describedby` didn't work on a select box for Narrator and I believe it might not work for JAWS either